src/Orleans/IDs/ActivationAddress.cs

```diff
@@ -32,6 +33,7 @@ internal class ActivationAddress
         public GrainId Grain { get; private set; }
         public ActivationId Activation { get; private set; }
         public SiloAddress Silo { get; private set; }
+        public MultiClusterStatus Status { get; private set; }
```

Comment:
Unless of course you explicitly do not want this extra field to be serialized, and then you need to mark it as `NonSerialized`.